src/Symfony/Bridge/PhpUnit/composer.json

```
@@ -18,14 +18,13 @@
         }
     ],
     "require": {
-        "php": ">=7.2.5 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
+        "php": ">=8.1.0 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
         "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
-        "php": ">=7.2.5"
+        "php": ">=8.1.0"
     },
     "require-dev": {
         "symfony/deprecation-contracts": "^2.5|^3.0",
-        "symfony/error-handler": "^6.4|^7.0|^8.0",
-        "symfony/polyfill-php81": "^1.27"
+        "symfony/error-handler": "^6.4.3|^7.0.3|^8.0"
     },
     "conflict": {
         "phpunit/phpunit": "<7.5|9.1.2"
```

You've removed compat classes for PHPUnit 7, haven't you?

---

I did, but the conflict still holds, isn't it?

---

you need to update the conflict to `<8.0` (or maybe `<8.5`) once we do not support PHPUnit 7.5 anymore.

---

> I did, but the conflict still holds, isn't it?

It's not strict enough anymore, is it?

```suggestion
        "phpunit/phpunit": "<8.5|9.1.2"
```

---

conflict updated to "phpunit/phpunit": "9.1.2"
we don't need ton conflict with v7 because it supports php ^7.1, so there no intersection anyway.

---

Ah, better: I removed the conflict line. We don't need to conflict with a random buggy version.

---

Makes sense.